root/layout/header/top.tt

```diff
@@ -25,7 +19,7 @@
             </ul>
         </li>
         <li class="data">
-            <a href="[% c.uri_for_action('/user/profile', [ c.user.name ]) %]">[% l('My Data') %]</a>
+            <span class="menu-header">[%- l('My Data') -%]</span>
```

Comment:
My feeling is that we should merge this into the other user menu, since they're kinda too related to be separate. That said, we still would need two lines of space for log in / create account, so we wouldn't win any real navbar space with that merge I guess.